en-US/Language.xml

```diff
@@ -1054,7 +1213,14 @@ Numbers (move from N entry) -->
 								</row>
 								 <row>
 									<entry> To create another administrator, click <guimenuitem>New</guimenuitem> on the <guimenuitem>File</guimenuitem> menu. </entry>
-									 <entry> To create another administrator account, click <guimenuitem>New</guimenuitem> on the <guimenuitem>File</guimenuitem> menu. <emphasis>OR</emphasis> To set privileges for another administrator, click <guimenuitem>New</guimenuitem> on the <guimenuitem>File</guimenuitem> menu. </entry>
+									 <entry> 
+									<para> 									 
+									 To create another administrator account, click <guimenuitem>New</guimenuitem> on the <guimenuitem>File</guimenuitem> menu. 
```

Comment:
@julian-cable This wording seems a little odd to me. Wouldn't we normally say "..., click File ---> New? 
I don't think this is critical, but it might be worth adding an example of this wording in a future edition of the style guide.

Reply:
Agree and updated in https://github.com/StyleGuides/WritingStyleGuide/pull/449.